cpp/harvest/marc_issn_lookup.cc

```diff
@@ -50,6 +50,47 @@ struct DebugInfo {
     DebugInfo() = default;
 };
 
+struct TitleInfo773 {
+    std::string title_, subfield_w_;
+    char ind1_, ind2_;
+
+    TitleInfo773() = default;
+
+    bool IsEqual(const TitleInfo773 &ti) const {
+        if ((title_ == ti.title_) && (ind1_ == ti.ind1_) && (ind2_ == ti.ind2_) && (subfield_w_ == ti.subfield_w_))
+            return true;
+
+        return false;
+    }
+
+    bool CompareButIgnoreW(const TitleInfo773 &ti) const {
```

Comment:
If you have functions doing similar things, it is always a good thing to name them in a similar way.
e.g.
IsEqual
CompareButIgnoreW => IsEqualWithoutW